jsk_fetch_robot/jsk_fetch_startup/config/config.bash

```diff
@@ -20,7 +20,7 @@ if [ $(hostname) = 'fetch15' ]; then
   export RS_SERIAL_NO_L515_HEAD="";
 
   export NETWORK_DEFAULT_WIFI_INTERFACE="wlan0";
-  export NETWORK_DEFAULT_ROS_INTERFACE="133.11.216.217";
+  export NETWORK_DEFAULT_ROS_INTERFACE="localhost";
```

Comment:
`NETWORK_DEFAULT_ROS_INTERFACE` can be removed.

Reply:
If do so, please also remove [here](https://github.com/knorth55/jsk_robot/blob/58a859cfd1f55a659acde7e381b14b4416325f9f/jsk_fetch_robot/jsk_fetch_startup/README.md?plain=1#L88-L90).

Reply:
I think we can leave it.

Reply:
uh, no, we should remove it.